utils/check_docstrings.py

```diff
@@ -79,7 +79,7 @@
     "AlbertTokenizerFast",
     "AlignTextModel",
     "AlignVisionConfig",
-    "AltCLIPTextConfig",
+    # "AltCLIPTextConfig",
```

Comment:
This should just be removed rather than commented out 

```suggestion
```

Reply:
Ok I will remove that. I guess I can do the same for VisionConfig too as they were in the same file.